pulp_gem/app/serializers.py

```diff
@@ -15,13 +15,18 @@
 from pulpcore.plugin.serializers import (
     ArtifactSerializer,
     MultipleArtifactContentSerializer,
-    SingleContentArtifactField,
     PublicationSerializer,
     PublicationDistributionSerializer,
     RemoteSerializer,
     RepositorySerializer,
 )
 
+try:
+    from pulpcore.plugin.serializers import SingleContentArtifactField
+except ImportError:
+    from pulpcore.app.serializers import SingleContentArtifactField
```

Comment:
@bmbouter Maybe i shouldn't do this. But you probably removed that class from the api for a reason...